.github/workflows/main.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-unit:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v1
+
+      - name: Install composer dependencies
+        run: composer install --prefer-dist --ignore-platform-reqs
+
+      - name: Run PHPUnit Integration tests
+        run: php7.4 ./vendor/bin/phpunit --testsuite=integration
```

Comment:
Could you use [act](https://github.com/nektos/act) to test if it works locally? On my machine I am getting `/github/workflow/1: line 2: composer: command not found`.

You can look at working workflows in other of our projects, for example here:

https://github.com/nextgen-tech/ewus-php/blob/e8820848f99c10fe075ccbe53f0b89ec32966ec4/.github/workflows/main.yml